Scripts/Runtime/Data/VirtualData/Jobs/VDJobWriter.cs

```diff
@@ -0,0 +1,84 @@
+using Anvil.Unity.DOTS.Jobs;
+using Unity.Collections;
+using UnityEngine;
+
+namespace Anvil.Unity.DOTS.Data
+{
+    /// <summary>
+    /// A struct to be used in jobs that is for writing new <typeparamref name="TInstance"/> to
+    /// <see cref="VirtualData{TKey,TInstance}"/>.
+    ///
+    /// Commonly used to add new instances. 
+    /// </summary>
+    /// <typeparam name="TInstance">The type of instance to add</typeparam>
+    [BurstCompatible]
+    public struct VDJobWriter<TInstance>
+        where TInstance : struct
+    {
+        private const int DEFAULT_LANE_INDEX = -1;
```

Comment:
Is it valid to use a lane index of -1? If not, then this should probably be `UNSET_LANE_INDEX`
(Same goes for any other places in this PR `DEFAULT_*` was used.)

Reply:
Yep good point, fixed.